src/meta/mod.rs

```diff
@@ -9,8 +9,10 @@ mod locale;
 pub mod name;
 mod owner;
 mod permissions;
+mod permissions_or_attributes;
 mod size;
 mod symlink;
+mod windows_attributes;
```

Comment:
how about `#[cfg(windows)]` here, and drop the `#cfg` inside `windows_attributes`?